airsenal/framework/utils.py

```diff
@@ -1318,6 +1319,32 @@ def get_recent_minutes_for_player(
     return minutes or [0]
 
 
+def was_historic_absence(player, gameweek, season, dbsession=None):
+    """
+    For past seasons, query the Absence table for a given player and season,
+    and see if the gameweek is within the period of the absence.
+
+    Returns: bool, True if player was absent (injured or suspended), False otherwise.
+    """
+    if season == CURRENT_SEASON:
+        # we only consider past seasons here.
+        return False
+    if not dbsession:
+        dbsession = session
+    absence = (
+        dbsession.query(Absence)
+        .filter_by(season=season)
+        .filter_by(player=player)
+        .filter(Absence.gw_from <= gameweek)
+        .filter(Absence.gw_until >= gameweek)  # should this be > rather than >= ?
```

Comment:
Nice work Nick! Re this comment - I've wondered about it in the past. Maybe the fairest (i.e. closest to playing live) way might be to use `<` and `>` rather than `<=` and `>=`? That would mean assuming we don't know about injuries until the gameweek after they start/finish?